packages/babel-plugin-transform-nullish-coalescing-operator/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "babel-plugin-transform-nullish-coalescing-operator",
+  "version": "7.0.0-beta.3",
+  "description": "Remove nullish coalescing operator",
+  "repository": "https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-nullish-coalescing-opearator",
+  "license": "MIT",
+  "main": "lib/index.js",
```

Comment:
could this just be `lib`?

Reply:
Yeah. I just copied it from a similar package.

Reply:
(reminder)